$OpenBSD: patch-configure,v 1.5 2016/01/25 21:33:15 sebastia Exp $

do not link against gnustep-gui in case its installed
decouple version of sope from the shared library interface versions
remove unused ssl check that generates a make variable with whitespace in
its name

--- configure.orig	Mon Jan 18 17:36:15 2016
+++ configure	Fri Jan 22 08:51:29 2016
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 #
 # Note: When adding make options to this script, ensure that the source still
@@ -370,9 +370,9 @@ genConfigMake() {
     cfgwrite "FHS_INSTALL_ROOT:=$ARG_PREFIX"
     cfgwrite ""
     cfgwrite "SOPE_SYSLIBDIR=\${DESTDIR}\${FHS_INSTALL_ROOT}/\$(CGS_LIBDIR_NAME)"
-    cfgwrite "SOPE_LIBDIR=\${SOPE_SYSLIBDIR}/sope-\${MAJOR_VERSION}.\${MINOR_VERSION}"
+    cfgwrite "SOPE_LIBDIR=\${SOPE_SYSLIBDIR}/sope-\${SOPE_MAJOR_VERSION}.\${SOPE_MINOR_VERSION}"
     cfgwrite "SOPE_SYSSHAREDIR=\${DESTDIR}\${FHS_INSTALL_ROOT}/share"
-    cfgwrite "SOPE_SHAREDIR=\${SOPE_SYSSHAREDIR}/sope-\${MAJOR_VERSION}.\${MINOR_VERSION}"
+    cfgwrite "SOPE_SHAREDIR=\${SOPE_SYSSHAREDIR}/sope-\${SOPE_MAJOR_VERSION}.\${SOPE_MINOR_VERSION}"
     cfgwrite "SOPE_DBADAPTORS=\${SOPE_LIBDIR}/dbadaptors"
     cfgwrite "SOPE_PRODUCTS=\${SOPE_LIBDIR}/products"
     cfgwrite "SOPE_SAXDRIVERS=\${SOPE_LIBDIR}/saxdrivers"
@@ -406,6 +406,7 @@ genConfigMake() {
     cfgwrite "debug:=no"
   fi
   cfgwrite ""
+  cfgwrite "NEEDS_GUI=no"
   
   if test $ARG_WITH_STRIP = 1; then
     cfgwrite "# configured to produce stripped code";
